bundles/org.eclipse.ui.editors/src/org/eclipse/ui/internal/editors/text/TextEditorMessages.properties

```diff
@@ -17,6 +17,8 @@ EditorsPlugin_internal_error=Internal Error
 
 TextEditorPreferencePage_displayedTabWidth=Displayed &tab width:
 TextEditorPreferencePage_lineSpacing=Line &spacing (extra % of font height):
+TextEditorPreferencePage_useFindReplaceOverlay=Use find/replace overlay
```

Comment:
It would be good to assign a shotcut value. I see that "y" seems to be unused in that page, so writing "overla&y" could do the trick.